.github/workflows/pr-open-terraform-test-env.yaml

```diff
@@ -17,88 +17,106 @@ jobs:
+        run: . venv/bin/activate
+
+      - name: Display Pip Version and confirm environment empty
+        run: |
+          pip -V
+          pip freeze
+
+      - name: Cache Python dependencies
+        uses: actions/cache@v2.1.4
+        env:
+          cache-name: cache-pip
+        with:
+          path: ~/.cache/pip
+          key: ${{ matrix.os }}-${{ matrix.python-version }}-build-${{ env.cache-name }}-${{ hashFiles('requirements*.txt') }}
+          restore-keys: |
+            ${{ matrix.os }}-${{ matrix.python-version }}-build-${{ env.cache-name }}-
+
+      - name: Install Dev Dependencies
+        run: pip install -r requirements-dev.txt
+
+      - name: Log full installed packages
+        run: pip freeze
+
+      - name: 'Terraform setup'
+        uses: hashicorp/setup-terraform@v1
+        with:
+          terraform_version: 0.14.7
+          terraform_wrapper: false
+
+      - name: 'Create Environment'
+        id: tfenv
+        run: python ./.github/scripts/tf-deploy-env.py --workspace modl --no-color
+
+      - name: 'Terraform plan dev output'
+        uses: actions/github-script@v3.1
+        env:
+          PLAN: "terraform\n${{ steps.tfenv.outputs.stdout }}"
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            const output = `
+            <details><summary>Show Plan</summary>
+
+            \`\`\` terraform
+            ${process.env.PLAN}
+            \`\`\`
+
+            </details>
+
+            Pusher: @${{ github.actor }}`;
+            github.issues.createComment({
+              issue_number: context.issue.number,
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              body: output
+            })
+
+      - name: 'Terraform apply'
+        run: python ./.github/scripts/tf-deploy-env.py --workspace "iac-${{ github.event.pull_request.number }}" --apply
+
+      - name: Setup SSH
+        env:
+          SSH_AUTH_SOCK: /tmp/ssh_agent.sock
+        run: |
+          KEY=$( aws secretsmanager get-secret-value --secret-id arn:aws:secretsmanager:us-east-1:077346344852:secret:db-ec2-key-pair-aofYm8  | jq --raw-output '.SecretString' )
+          DB_PASSWORD=$( aws secretsmanager get-secret-value --secret-id arn:aws:secretsmanager:us-east-1:077346344852:secret:db-creds-WszNCl  | jq --raw-output '.SecretString' | jq -r .password )
+          DB_USERNAME=$( aws secretsmanager get-secret-value --secret-id arn:aws:secretsmanager:us-east-1:077346344852:secret:db-creds-WszNCl  | jq --raw-output '.SecretString' | jq -r .username )
+          DB_HOST=$( aws rds describe-db-cluster-endpoints --filters="Name=db-cluster-endpoint-type, Values=writer" | jq --raw-output ".[][].Endpoint" )
+
+          ssh-agent -a $SSH_AUTH_SOCK > /dev/null
+          ssh-add - <<< "${KEY}"
+
+          scp -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -r ./.github/schema/schema.sql ec2-user@ec2-54-226-191-183.compute-1.amazonaws.com:/tmp
```

Comment:
currently hard coding the ec2 instance endpoint and the database endpoint. The code in line 116 works for me locally with my aws credentials and I can add more filters for the specific PR number but it'll pick up all the writer instances for now if we have more than one cluster up.
so just add your new instance endpoint once the db gets spun up.

Reply:
we'll need a different jump host for prod/modl

Reply:
yeah sorry I meant db instance.. the ec2 is static for each environment.. any thoughts on alternative way of getting the db endpoint besides hard coding it in? I guess I was just thinking that if we had multiple PRs open, it wouldn't auto pul your corresponding db instance, it would just pull the last instance of all the clusters.. but probably wouldn't be an issue with modl/prod